### ISSUE TITLE
[Wave 6: Bottom Up] Test large animation in the app

### ISSUE THREAD

**Issue by puneetlath** [Expensify/App#30600]
We want to add a video or animation to the FAB when it is first opened. We don't yet support in-app video playback, so animation is our only currently option. I've heard that there are concerns about animations over a certain size/length. But also that we now support .lottie files which reduces animation file sizes by a lot. 

Could we test this animation in the app and see how it performs? 

Here is the full 45 second animation: 
[inProductExplainer-Mobile-v2.json (1).zip](https://github.com/Expensify/App/files/13208267/inProductExplainer-Mobile-v2.json.1.zip)

Here is one broken in to three 15 second portions: 
[InProductExplainer-JSONTest.zip](https://github.com/Expensify/App/files/13208081/InProductExplainer-JSONTest.zip)

We'd ultimately like it to live in the Global Create menu and only be shown when the global create menu is open by default, but we can start out by testing it anywhere. 

![Untitled](https://github.com/Expensify/App/assets/1371996/6a9ed6cf-8bce-4703-904a-4cff3fd77ef0)

![image (26)](https://github.com/Expensify/App/assets/1371996/5a4aaedc-9e83-4a45-aee4-db2f128ed028)

cc @roryabraham

**Comment by kosmydel** [comment]
Hey, I would love to help you here. Can you send me some large animations? I will check next week.

**Comment by kosmydel** [comment]
Hey, I would love to help you here. Can you send me some large animations? I will check next week.

**Comment by puneetlath** [comment]
Are you able to use the two zip files linked above?

**Comment by kosmydel** [comment]
> Thanks so much! Are you able to use the two zip files linked above?

Sure! I haven't noticed it before.

**Comment by kosmydel** [comment]
Hey, I used [this converter](https://lottiefiles.com/tools/lottie-to-dotlottie) to convert animations to the `.lottie` format, but it looks like some of the animations are broken, and couldn't be converted.

I was able to test it on `ChatFlow`. Here is a video:

https://github.com/Expensify/App/assets/104823336/a1e11735-9547-45a8-a04a-37eaf059eb5b

[Here is a link to the branch](https://github.com/software-mansion-labs/expensify-app-fork/tree/%40kosmydel/dot-lottie-performance-test), where you can test it (just go to Settings -> Security).

Can you send me animations in `.lottie` format to test it? Or do you want me to test it in `.json` format (which is about 10x larger in size).

**Comment by puneetlath** [comment]
>Can you send me animations in .lottie format to test it? Or do you want me to test it in .json format (which is about 10x larger)?

So dumb question, I can't actually figure out how we get a file in a .lottie format. Can a file be exported from After Effects into this format? I'm seeing guides [like this](https://ampliphibiz.com/how-to-create-lottie-files-in-after-effects/), but they still seem to all mention that the exported file will be in JSON format.

**Comment by kosmydel** [comment]
Hey @puneetlath,
I found a plugin for After Effects [here](https://lottiefiles.com/plugins/after-effects). They say that it offers "dotLottie export support". Can you confirm that it works/doesn't work?

If it doesn't work, then we probably had to rely on the converter [here](https://lottiefiles.com/tools/lottie-to-dotlottie).

**Comment by puneetlath** [comment]
Ok thanks! I'll see if they can use that plugin to export as .lottie

**Comment by puneetlath** [comment]
Ok here are two new ones. They installed that plugin, but weren't able to export as .lottie. On smaller animations, they were able to export with the plugin, but not this bigger one. So maybe it's just too big? 

So they created the .lottie using the converter. But in any case let's try both of these out. If we find that they don't work, then we'll have to go the video route. 

[InProductExplainer-Mobile-v16.lottie.zip](https://github.com/Expensify/App/files/13299961/InProductExplainer-Mobile-v16.lottie.zip)

[InProductExplainer-Mobile-v16.zip](https://github.com/Expensify/App/files/13299960/InProductExplainer-Mobile-v16.zip)

**Comment by kosmydel** [comment]
Hey,
I've tried both `.lottie` and `.json`. Even their official preview doesn't open those animations ([link here](https://app.lottiefiles.com/preview)). If it works in this preview, it will (in 99%) work in our app.

My guess is it is something related to the videos/images attached to this animation. After decompression I can't even open it on my Mac. So those might be corrupted.

It might be a good idea to report it as a bug [here](https://help.lottiefiles.com/hc/en-us/requests/new?ticket_form_id=22196929106713). 
I can do this, if you provide me with exact steps how you generate the `.lottie` file (probably the source file of the animation in After Effects would be good to send them), so I can provide them with as many details as possible.

**Comment by puneetlath** [comment]
Ok good to know! I'll ask for the steps they used to create the file. I've also DM'd you the pre-compression .lottie file in case that makes a difference.

**Comment by kosmydel** [comment]
> Ok good to know! I'll ask for the steps they used to create the file. I've also DM'd you the pre-compression .lottie file in case that makes a difference.

I've checked it, and the error still occurs. I'm testing it [here](https://app.lottiefiles.com/preview).

After I decompressed the `.lottie` file, the images attached are broken (cannot open it on my Mac). It might be the reason why it doesn't work.
![Screenshot 2023-11-13 at 11 36 55](https://github.com/Expensify/App/assets/104823336/fb1f029c-d77e-4311-b95b-882386104515)
